src/main/java/com/softserveinc/ita/steps/TestsSteps.java

```diff
@@ -3,20 +3,23 @@
 import com.softserveinc.ita.models.TestEntity;
 import com.softserveinc.ita.pageobjects.LoginPage;
 import com.softserveinc.ita.pageobjects.admin.TestsPage;
+import com.softserveinc.ita.pageobjects.modals.DeletingFormModal;
 
-import static com.softserveinc.ita.util.DataProvider.*;
+import static com.softserveinc.ita.util.DataProvider.ADMIN_LOGIN;
+import static com.softserveinc.ita.util.DataProvider.ADMIN_PASSWORD;
 
 public class TestsSteps {
+    private TestsPage page;
 
     public TestsPage openTestsPage(String subject) {
```

Comment:
stop returning anything by methods in step classes, init page variables here, but try to perform all operations in step class if needed, asserts also could be performed here